src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -672,4 +837,168 @@ public static String replaceCodes(String text, char from) {
     public static String replaceCodes(String text, char from, char to) {
         return factory.replaceLegacyCodes(text, from, to);
     }
+
+    /**
+     * Helper class that allows easier/chained calls to the format method
+     * without extra steps
+     */
+    public static final class FormatBuilder implements ResettableBuilder<FormatBuilder> {
```

Comment:
It is completly free from its context.
This util class is just a helper to craft the context together, but it has nothing to do with the templates (except calling the format call in the end.)
It was meant to be a simplification for
````java
Map<String,Object> data = new HashMap<>();
data.put("foo", foo);
data.put("bar", bar);
Texts.format(template, data);
````
simplied into:
````java
Texts.prepare(tempalte).with("foo", foo).with("bar", bar).format();
````
This becomes increasingly in handy with longer values to set (IMO the "NEW" example in the description shows its use best)